app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -278,11 +278,11 @@
     <div class="ui section divider"></div>
     <div class="field">
       <h4>{{t 'Choose payment methods'}}</h4>
-      {{#if canAcceptPayPal}}
```

Comment:
This is different. `canAcceptPayPal ` tells if the currency selected by user is supported by PayPal. So, this and the related computed property is needed.